system/Autoloader/Autoloader.php

```diff
@@ -124,7 +124,7 @@ public function initialize(Autoload $config, Modules $modules)
             $this->files = $config->files;
         }
 
-        if (isset($config->helpers)) {
+        if ($config->helpers !== []) {
```

Comment:
this needs to be verified first whether property exists in the beginning or intruduced later, otherwise, this change should be skipped and need to land in 4.6, with upgrade guide updated.

Reply:
Look above - all arrays from the config are also checked. What kind of verification are we talking about?

Reply:
The property named helper in the config, since some minor version intruduce new property, isset will silently check, while direct property call may cause undefined property

Reply:
Ok. Revert

Reply:
Ah, rector say:
```
Property Config\Autoload::$helpers (array<int, string>) in isset() is not nullable.  
         🪪  isset.property 
```

Reply:
you mean phpstan, you can regenerate baseline for that

Reply:
helpers property seems introduced in 4.3.0 of appstarter autoload

https://github.com/codeigniter4/appstarter/compare/v4.2.12...v4.3.0

while it should surpass many upgrade process, I prefer this isset() change to direct property call to go to 4.6.0 :)